ksqldb-execution/src/main/java/io/confluent/ksql/execution/codegen/helpers/NullSafe.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2020 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.execution.codegen.helpers;
+
+import java.util.function.Function;
+
+/**
+ * Helper functions around null handling
+ */
+public final class NullSafe {
```

Comment:
Unit tests? Seems useful to know what the returned code is.

Reply:
Likewise.